File: backend/src/main/java/com/java/NBE4_5_1_7/domain/news/entity/NewsItem.java
package com.java.NBE4_5_1_7.domain.news.entity;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Getter
@Setter
public class NewsItem {
    @JsonProperty("title")
    private String title;
    @JsonProperty("originallink")
    private String originallink;
    @JsonProperty("link")
    private String link;
    @JsonProperty("description")
    private String description;
    @JsonProperty("pubDate")
    private String pubDate;
}

File: backend/src/main/java/com/java/NBE4_5_3_7/domain/news/entity/NewsItem.kt
package com.java.NBE4_5_3_7.domain.news.entity

import com.fasterxml.jackson.annotation.JsonProperty

class NewsItem {
    @JsonProperty("title")
    var title: String? = null

    @JsonProperty("originallink")
    var originallink: String? = null

    @JsonProperty("link")
    var link: String? = null

    @JsonProperty("description")
    var description: String? = null

    @JsonProperty("pubDate")
     var pubDate: String? = null
}
